schema/artist/index.js

```diff
@@ -1,5 +1,7 @@
 import _ from 'lodash';
 import Image from '../image';
+import ArtworkSorts from '../sorts/artwork_sorts'
```

Comment:
I think this is fine. The other alternative would be organizing them under the type: like /artwork/sorts & /partner_shows/sorts